test/indexer/test_hca_indexer.py

```diff
@@ -1777,33 +1777,45 @@ def test_related_files_field_exclusion(self):
                                      })
         self.assertEqual(0, hits["hits"]["total"])
 
-    def test_samples_downstream_entities(self):
+    def test_downstream_entities(self):
         """
-        Verify that samples include bam files from stitched subgraphs
+        Verify that samples and cell_suspensions include analysis files from
+        stitched subgraphs
         """
         bundle = self.bundle_fqid(uuid='79fa91b4-f1fc-534b-a935-b57342804a70',
                                   version='2020-12-10T10:30:00.000000Z')
         self._index_canned_bundle(bundle)
 
-        def is_sample_aggregate(hit):
-            entity_type, aggregate = self._parse_index_name(hit)
-            return entity_type == 'samples' and aggregate
+        def get_aggregates(hits, type):
+            for hit in hits:
+                entity_type, aggregate = self._parse_index_name(hit)
+                if entity_type == type and aggregate:
+                    yield hit['_source']['contents']
 
-        samples = [
-            hit['_source']['contents']
-            for hit in self._get_all_hits()
-            if is_sample_aggregate(hit)
-        ]
+        all_hits = self._get_all_hits()
+        samples = list(get_aggregates(all_hits, 'samples'))
         self.assertEqual(15, len(samples))
 
-        for sample in samples:
+        def assert_analysis_files(hit):
             analysis_file_formats = {
                 file['file_format']
-                for file in sample['files']
+                for file in hit['files']
                 if 'DCP/2 Analysis' in file['file_source']
             }
             self.assertEqual({'bam', 'loom'}, analysis_file_formats)
 
+        for sample in samples:
+            assert_analysis_files(sample)
+            self.assertGreater(len(sample['donors']), 0)
+            self.assertGreater(len(sample['specimens']), 0)
+
+        cell_suspensions = list(get_aggregates(all_hits, 'cell_suspensions'))
+        self.assertEqual(22, len(cell_suspensions))
+        for cell_suspension in cell_suspensions:
+            assert_analysis_files(cell_suspension)
+            inner = one(cell_suspension['cell_suspensions'])
+            self.assertGreater(inner['total_estimated_cells'], 0)
```

Comment:
```suggestion
            self.assertEqual(null_int.null_int, inner['total_estimated_cells'])
```
All cell suspensions in this bundle omit the `estimated_cell_count` field, so each `contents.cell_suspension.total_estimated_cells` is the translated null value.

Reply:
> Also, any thoughts on #3281 (comment) :

That's what the `drop!` commit is for, running the integration tests with the bundles that initially caused #2868.

> each contents.cell_suspension.total_estimated_cells is the translated null value

Nice catch. That might be a problem since I expect Hannes wants these tests to prove that the cell counts are non-zero (since that's what the ticket is about). topic for parking lot.

Reply:
```suggestion
            self.assertEqual(inner['total_estimated_cells'], null_int.to_index(None))
```
The test bundle doesn't contain any cell suspensions with a cell count field (`estimated_cell_count` or the deprecated `total_estimated_cells`), so the value in the index is a translated `None`.

Reply:
@hannes-ucsc: "First, translate the field values using filed types so that the field replacement value becomes null. Secondly, since the stitched on cell suspensions in this bundle lack estimated_cell_count, add that programmatically before indexing the test bundle (during the test)."